lgsm/data/serverlist.csv

```diff
@@ -46,6 +46,7 @@ kf2,kf2server,Killing Floor 2
 l4d,l4dserver,Left 4 Dead
 l4d2,l4d2server,Left 4 Dead 2
 mc,mcserver,Minecraft
+mh,mordhauserver,Mordhau
```

Comment:
maybe better calling it mhserver, as naming convention

Reply:
Changed the name now